### PR TITLE
List with checkbox

### PR DESCRIPTION
**Pull request by Mucyosoda**
This project is designed to display a list of activities to be fulfilled in a day.
 I made this with different technology like Javascript, HTML, CSS, linters, and webpack.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import './style.css';
+
```
Comment (rloterh) [inline src/index.js:2]: - [ ] Your index.js does not have any new JavaScript files imported as modules. Kindly ensure that your `index.js` has one new JavaScript file imported as a module, which should contain methods related to the `status updates` (completed: true / false).

Comment (rloterh) [inline src/index.js:2]: - [ ] There's no implemented event listener. Kindly ensure that there is an event listener on the checkbox so as to trigger an event whenever the checkbox is checked or unchecked.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mucyo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — COMMENTED** [review]
Hi,

Good job so far! 👏🏼 @Mucyosoda 


## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 - N/A.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)Hi,

**Pull request merged**